### DISCUSSION TITLE
Timelimit on the config task

### DISCUSSION THREAD

**Discussion by venkycr** [ds4dm/ml4co-competition#56] [Q&A] [answered]
For the evaluation of the test datasets, in the config task, is there a timelimit set?

The optimization can take a long time to execute. The longer an optimization goes, the larger the (negative) cumulative reward. So I am assuming there is a timelimit to ensure there is a fair comparison between two solutions. Is that the right assumption?

**Comment by gasse** [comment, marked as answer]
Hi @venkycr ,

There is indeed a time limit set, in all three tasks. We've chosen to use the following in each task:
- primal: 5 minutes
- dual: 15 minutes
- config: 15 minutes

See the details of each task (challenge) [here](https://www.ecole.ai/2021/ml4co-competition/#challenges).

Best,
Maxime

**Reply by venkycr** [reply]
Thank you
